src/main/java/lotto/controller/LottoApp.java

```diff
@@ -1,19 +1,23 @@
 package lotto.controller;
 
 import lotto.domain.Amount;
+import lotto.domain.Lotto;
 import lotto.domain.LottoGame;
 import lotto.domain.Winning;
 import lotto.view.InputView;
 import lotto.view.ResultView;
 
+import java.util.List;
+
 public class LottoApp {
     public static void main(String[] args) {
-        Amount purchaseAmount = new Amount(InputView.inputPurchaseAmount());
+        final Amount purchaseAmount = Amount.of(InputView.inputPurchaseAmount());
+        final Amount manualLottoCount = Amount.of(InputView.inputManualLottoCount());
 
-        final LottoGame lottoGame = new LottoGame(purchaseAmount);
+        List<String> manualLottoTexts = InputView.inputManualLotto(manualLottoCount);
```

Comment:
```suggestion
        Lotto manualLottoTexts = Lotto.of(InputView.inputManualLotto(manualLottoCount));
```
구조적으로 어떻게 되어야할지 모르겠지만,
저는 그냥 바로 객체화 해버릴것같아요 🤔